### PR TITLE
Update CoreWCF.Templates target runtimes

### PR DESCRIPTION
**Pull request by g7ed6e**
Add net7.0
Remove net5.0

### CODE REVIEW

File: azure-pipelines.yml
Comment (mconnew) [inline azure-pipelines.yml:13]: Did you mean TestTemplatesStage.yml

**Comment by mconnew** [comment]
LGTM. Only question is whether you specified the correct filename in the pipeline excludes. Feel free to merge as that won't need a second review if you modify it.

**Review by mconnew — APPROVED** [review]

File: azure-pipelines.yml
Comment (g7ed6e) [reply azure-pipelines.yml]: Yes I pushed a fix. thanks!

**Pull request merged**
